tests/validator_test.go

```diff
@@ -408,7 +409,21 @@ var _ = Describe("Template validator", func() {
 		It("[test_id:2960] Negative test - Create a VM with machine type violation", func() {
 			template = TemplateWithRules()
 			Expect(apiClient.Create(ctx, template)).ToNot(HaveOccurred(), "Failed to create template: %s", template.Name)
+			// set value unfulfilling validation
+			vmi = addDomainResourcesToVMI(vmi, 2, "test", "128M")
+			vm = NewVirtualMachine(vmi)
+			vm.ObjectMeta.Annotations = map[string]string{
+				TemplateNameAnnotation:      template.Name,
+				TemplateNamespaceAnnotation: template.Namespace,
+			}
+			eventuallyFailToCreateVm(vm)
+			Expect(errors.IsInvalid(apiClient.Create(ctx, vm))).To(BeTrue(), "Should match error type because of unfulfilled validations")
```

Comment:
This `Expect()` is not needed. The same check is already done in `eventuallyFailToCreateVm(vm)`

Reply:
this is git :/ I did not really change this code at all, it just got confused because of the similarity with the function I did add below... I think changing this is for a separate PR.

Reply:
So the line was added by mistake during rebase.

Can you rebase this PR on the latest master and remove this line?

Reply:
you are right! done.